stories/story-utils/storeSetup.ts

```diff
@@ -1,61 +1,133 @@
-import {IRootState} from '@/store'
 import Vue from 'vue'
 import {namespace} from 'vuex-class'
 const flowVuexNamespace = namespace('flow')
 const builderVuexNamespace = namespace('builder')
-import {findFlowWith, IBlock, IContext, IFlow} from '@floip/flow-runner'
+import {
+  IBlock,
+  IFlow,
+  ILanguage,
+  SupportedContentType,
+  SupportedMode
+} from '@floip/flow-runner'
 import {get, isEmpty, cloneDeep} from 'lodash'
+import { IResourceDefinitionVariantOverModesFilter } from "../../src/store/flow/resource";
+import Component from 'vue-class-component'
+import caseBlockStore, {BLOCK_TYPE as CASE_BLOCK_TYPE} from '@/store/flow/block-types/Core_CaseBlockStore'
 
 let storyInitState: any = {}
 
 /**
- * Safe register block module
- * Because some weird race condition is leading to modules not getting unregistered when clicking between stories before the next story re-registers
+ * Vue class used to gather required Getter, Mutation, Action for the BaseMounted binding
  */
-export const safeRegisterBlockModule = async function(BLOCK_TYPE: string, blockTypeStore: IRootState) {
-  // @ts-ignore - TS2551: Property '$store' does not exist on type
-  if (this.$store.hasModule(['flow', BLOCK_TYPE])) {
-    // @ts-ignore - TS2551: Property '$store' does not exist on type
-    this.$store.unregisterModule(['flow', BLOCK_TYPE])
+@Component<any>({})
```

Comment:
@jcbashdown , If you remember we couldn't extends class correctly, this was the solution. We have to add `@Component` decorator for Parent & Child classes

Reply:
Do you remember exactly what the issue was? I have extended a flow-builder class with a custom template in voto5 but had some weird behaviour. I had to define a proxy between activeFlow on the child and the parent as even though I could see the computed prop on the child in vue devtools I couldn't actually access the value correctly:

https://bitbucket.org/voto/voto5/pull-requests/1787/vmo-3120-flow-lifecycle#Lresources/assets/js/floip/builder/views/CustomNewFlow.vueT47

Reply:
@jcbashdown : the pbm was we couldn't access correclty the parent's props & methods from its children.

Thanks to StackOverFlow, someone has replied to my question and it seems working :+1:
https://stackoverflow.com/questions/63004313/vue-class-component-inheritance-method-from-base-class-is-not-a-function/66194360#66194360

Reply:
Only just realised you asked the question. Nice!